### DISCUSSION TITLE
Storybook is not running with yarn

### DISCUSSION THREAD

**Discussion by bikashghimire** [storybookjs/storybook#24744] [Help]
<img width="834" alt="Screenshot 2023-11-07 at 14 31 19" src="https://github.com/storybookjs/storybook/assets/36932308/b4231578-8350-40c3-8e79-74970919b6f1">

**Comment by jonniebigodes** [comment]
@bikashghimire thanks for taking the time to reach out to us with your issue. So that we maintainers and the community may help you solve your issue, can you provide us a bit more context about your environment (e.g., OS, package manager) so that it can be looked at? Or as you may or may not know, we just recently released a new version (7.6). Would you be interested in upgrading the version checking if the issue still persists, and reporting back to us?

Looking forward to hearing from you.

Hope you have a great day.

Stay safe

**Reply by bikashghimire** [reply]
Hello @jonniebigodes, appreciate your response. It was Mac environment.  It was challenging for me initially, but eventually, it began functioning.  but I have to run the program command `yarn storybook` twice so that it works. I havenot upgraded yet but I plan to do so in the future.
